WTWSMS/history/titles/c_valladolid.txt

```diff
@@ -2,29 +2,18 @@
 	holder=8607
 	liege = "k_visigoths"
 }
-510.1.1={
-	holder=8608
+507.1.1={
+    holder = 13100026 #Duratus Sappus
+	liege = "d_sappia"
```

Comment:
Is Sappia supposed to be independent? Then you should use `liege = 0` here.

Reply:
It is supposed to, yes, I will change it.

Reply:
This should still be solved.

Reply:
```suggestion
	liege = 0
```